tests/Makefile

```diff
@@ -55,7 +55,7 @@ DROP_DB_CMD = bash -c 'psql -h localhost \
 		-p $${TEST_DBPORT} -U $${TEST_DBUSERNAME} postgres \
 		-c "DROP DATABASE codechecker_config_$${CODECHECKER_DB_DRIVER}"'
 
-run_test: package venv_dev
+run_test: venv_dev
```

Comment:
Why did you remove to build the package before tests?

Reply:
I removed them because we need to run the internal tests with a different CodeChecker configuration. If the test targets always rebuild the package before each test we can not run the tests with a different config.

Reply:
In this case please update the documentation: https://github.com/Ericsson/codechecker/blob/master/tests/README.md

Reply:
done